index.css

```diff
@@ -0,0 +1,233 @@
+html {
+  box-sizing: border-box;
+  width: 100%;
+}
+
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: azure;
+}
+
+.nav {
+  height: 60px;
+  background-color: burlywood;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 80px;
+  padding-right: 80px;
+  border: 2px solid bisque;
+  margin: 0 0;
+}
+
+.nav a {
+  text-decoration: none;
+  font-size: 23px;
+  font-weight: 100;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  color: black;
+}
+
+.nav a:hover {
+  color: brown;
+}
+
+.nav-ul {
+  display: flex;
+  list-style: none;
+}
+
+.nav-ul li {
+  margin-left: 40px;
+  font-weight: 100;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  text-decoration: none;
+  color: black;
+  border-left: 2px solid black;
+  padding-left: 20px;
+}
+
+.clock-section {
+  font-weight: 100;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
```

Comment:
- [ ] Good job so far, but you should make sure you follow the given design. The displayed date should be to the right of the page, kindly fix that 😉 

Your design

<img width="721" alt="Screen Shot 2023-02-21 at 14 00 54" src="https://user-images.githubusercontent.com/74501523/220329267-2eb8bc51-516b-4046-80c1-6f7a0abc8e0c.png">

Given design

<img width="449" alt="Screen Shot 2023-02-21 at 14 01 33" src="https://user-images.githubusercontent.com/74501523/220329467-c871af72-77ca-4c31-99d0-e5f4819bcd56.png">